src/components/modules/moim-detail/ConfirmationJoinModal.tsx

```
@@ -32,48 +27,50 @@ const LoginButton = () => {
-      </DialogHeader>
-      <DialogFooter className="flex h-12 flex-row gap-2 sm:h-15">
-        <DialogClose asChild>
-          <Button
-            variant="outline"
-            className="h-full flex-1 rounded-[12px] border border-gray-100 text-base font-medium text-gray-500 shadow-none hover:cursor-pointer hover:bg-transparent hover:shadow-sm sm:rounded-2xl sm:text-lg"
-          >
-            취소
-          </Button>
+        <DialogClose asChild className="absolute top-6 right-6 rounded-full p-1">
+          <button aria-label="닫기">
+            <Image src="/icons/delete.svg" alt="닫기 버튼" width={24} height={24} />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)